specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -12906,7 +12921,7 @@
         ],
         "summary": "Gets a named operation for a network trace capturing (or deployment slot, if specified).",
         "description": "Gets a named operation for a network trace capturing (or deployment slot, if specified).",
-        "operationId": "WebApps_GetNetworkTraceOperationSlotV2",
+        "operationId": "WebApps_GetNetworkTraceOperationV2Slot",
```

Comment:
This is a breaking change that will require a major version bump in any published SDKs - are you sure this is necessary?

Reply:
Same as above.

Reply:
Reverted this change.

Reply:
@annatisch The PR has been updated. Please take a look.